### ISSUE TITLE
Create "dashboard on registration activities" on covid19.healthdata.be

### ISSUE THREAD

**Issue by JohanvanBussel** [Sciensano-Healthdata/HD-Corona#24]
it would be helpfull to have an overview of the following statistics:
- total number of submissons of the past day, by form
- total number of submissions, by day, by form
- highest number of submissons on a day since start registration, by form
- lowest number of submissons on a day since start registration, by form
- geographical representation of submissons past day, by form
- geographical representation of total number submissons since start registration, by form

**Comment by VrankenKris** [comment]
@neuronick @maerckaert I need the start datetime per message id to be able to do calculations of time needed to enter a registration. This can be given via another way then the normal data flow if this is faster

**Comment by neuronick** [comment]
the start / completed datetime are since a couple of days ago sent with the rest of the data.

**Comment by VrankenKris** [comment]
Yes, it has but not at the time of writing the comment (26th of March).
The data for the dashboard has not yet been created, because we have at the moment data flows to setup :-)
